factory/config.go

```diff
@@ -65,8 +65,10 @@ type Tls struct {
 }
 
 type Mongodb struct {
-	Name string `yaml:"name"`
-	Url  string `yaml:"url"`
+	Name           string `yaml:"name,omitempty"`
+	Url            string `yaml:"url,omitempty"`
+	AuthKeysDbName string `yaml:"authKeysDbName"`
+	AuthUrl        string `yaml:"authUrl"`
```

Comment:
Are we mandating these 2 new fileds to be mandatory present. If not then could we use same default Name & URL?

Reply:
Yes, If they do not exist, we will use same URL but another DB which  name is `authentication` by default.